crates/cast/src/main.rs

```diff
@@ -84,44 +95,45 @@ async fn main() -> Result<()> {
     update_cast_config(&mut config, &cli);
 
     let provider = get_provider(&config.rpc_url)?;
+    let mut account: Option<SingleOwnerAccount<&JsonRpcClient<HttpTransport>, LocalWallet>> = None;
+
+    match &cli.command {
+        Commands::Call(_) | Commands::Account(_) => {}
+        Commands::Multicall(value)
+            if matches!(
+                &value.command,
+                starknet_commands::multicall::Commands::New(_)
+            ) => {}
+        _ => account = Some(get_account(&cli, &config, &provider).await?),
+    }
 
     match cli.command {
         Commands::Declare(declare) => {
-            account_file_exists(&config.accounts_file)?;
-            let chain_id = get_chain_id(&provider).await?;
-            let mut account =
-                get_account(&config.account, &config.accounts_file, &provider, chain_id)?;
-
             let mut result = starknet_commands::declare::declare(
                 &declare.contract,
                 declare.max_fee,
-                &mut account,
+                &mut account.unwrap(),
```

Comment:
It is generlly not very good idea to use unwrap() in prod code, as this will panic if the Option is None, which can lead to runtime errors and crashes. 

Panics are not a graceful way to handle errors, and they can make an app unreliable. It is better to do something like:

```
 if let Some(whatever) = &mut account {
     (...)
 } else {
     (...)
}
```

or something like `match`. You can look through our codebase to see examples of that.

Reply:
Agree with everything. Decided to use unwrap here, since by the logic of the code flow, account shouldn't be `None` at this point in code.

Reply:
Sure, I understand. Still, I guess we should avoid using it if we can.